frontend/talentsearch/src/js/components/Browse/OngoingRecruitmentSection.tsx

```diff
@@ -0,0 +1,1191 @@
+        })}
+      </Heading>
+      <p data-h2-margin="base(0, 0, x1, 0)" data-h2-font-weight="base(700)">
+        {intl.formatMessage({
+          id: "2czUAZ",
+          defaultMessage:
+            "Not seeing an active recruitment that matches your skillset? No problem, we still want to hear from you.",
+          description: "summary for section with ongoing pool advertisements",
+        })}
+      </p>
+      <p>
+        {intl.formatMessage({
+          id: "BWvfeG",
+          defaultMessage:
+            "We also offer passive recruitment buckets that allow us to find talent fast when the demand arises. While there’s no guarantee a job will result from the opportunities below, it’s an easy way for your name and resume to be found by managers when the time comes. Feel free to submit your name to any bucket that matches your skills.",
+          description:
+            "instructions for section with ongoing pool advertisements",
+        })}
+      </p>
+      <div data-h2-padding="base(x2, 0, 0, 0) p-tablet(x3, 0, 0, 0)">
+        <Accordion.Root
+          data-h2-margin="base(0)"
+          data-h2-padding="base(0)"
+          type="multiple"
+        >
+          {dataModel.streams
+            .filter(
+              // filter streams with no classifications with a pool ID to apply to
+              (stream) =>
+                !!stream.classifications.find(
+                  (classification) => classification.poolAdvertisementId,
+                ),
+            )
+            .map((stream) => (
+              <Accordion.Item value={stream.key} key={stream.key}>
+                <Accordion.Trigger
+                  subtitle={stream.summary}
+                  data-h2-font-size="base(h4)"
+                >
+                  {stream.title}
+                </Accordion.Trigger>
+                <Accordion.Content>
+                  <div
+                    data-h2-display="base(grid)"
+                    data-h2-grid-template-columns="base(1fr) p-tablet(1fr 1fr)"
+                    data-h2-gap="base(x.5) p-tablet(x2)"
+                    data-h2-align-items="base(center)"
+                  >
+                    {stream.classifications
+                      .filter(
+                        // filter to only classifications with a pool ID that can be applied to
+                        (classification) => classification.poolAdvertisementId,
+                      )
+                      .map((classification) => (
+                        <div
+                          key={classification.title}
+                          data-h2-padding="base(0,0,x1, 0)"
+                        >
+                          <h3
+                            data-h2-font-size="base(h6)"
+                            data-h2-font-weight="base(700)"
+                            data-h2-padding="base(0,0,x0.75, 0)"
+                          >
+                            {classification.title}
+                          </h3>
```

Comment:
Similar to the previous comment, we should bump this down to a `<h4 />`

```suggestion
                          <h4
                            data-h2-font-size="base(h6)"
                            data-h2-font-weight="base(700)"
                            data-h2-padding="base(0,0,x0.75, 0)"
                          >
                            {classification.title}
                          </h4>
```

This is to the page heading structure is:

- h1: Browse IT jobs
    - h2: Active talent recruitment processes
    - h2: Apply to ongoing recruitment
        - h3: IT business line advisory services
            - h4: Level 2 (Analyst)

Reply:
Fixed in [fix headings](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5043/commits/2c7be8d62a59a75ee3df90fda881f7d406b3346e)